### ISSUE TITLE
Sprinting + jumping on ice triggers rubberbanding after ~25 blocks

### ISSUE THREAD

**Issue by MarkElf** [NoCheatPlus/Issues#460]
#### Short description of the issue.
Jumping and sprinting for about 20-25 blocks on ice with blocks above causes the player to rubberband. I thought this issue was resolved before, seems that it wasn't (#57). May have something to do with the 15% increased speed cap if I had to take a guess.

#### Complete output of the _ncp version_ command (please prefer text here).
17-06-14 13:09:01 [INFO] ---- Version information ----
#### Server ####
git-Spigot-21fe707-e1ebe52 ~MC: 1.8.8~
  detected: 1.8.8
#### NoCheatPlus ####
Plugin: 3.15.2-SNAPSHOT-sMD5NET-b1113
MCAccess: 1.8.4-1.8.8 / Spigot-CB-1.8_R3
Features:
  blocks: BlocksMC1_4 | BlocksMC1_5 | BlocksMC1_6_1 | BlocksMC1_7_2 | BlocksMC1_8
  checks: FastConsume | Gutenberg | AttackFrequency | FlyingFrequency | KeepAliveFrequency
  defaults: pvpKnockBackVelocity
  packet-listeners: UseEntityAdapter | MovingFlying | OutgoingPosition | KeepAliveAdapter | SoundDistance | CatchAllAdapter
Hooks: AllViolations~NCP~ 1.0
#### Related Plugins ####
ProtocolLib v4.2.1 (also reproduced with ProtocolLib v4.2.0, if that has any impact)

#### How to reproduce the issue (is it random / always happens / side conditions)?
Triggers every 25 or so blocks.

#### Have you made any (noteworthy) changes to the default configuration of NCP?
Default configuration.

#### Do you run any other special plugins that affect game mechanics next to NC+ (adding skills, machines, adding/changing blocks, other anti cheating)?
None.

#### Provide additional information, such as error/debug/violation logs. (For reproducible issues with moving consider: https://github.com/NoCheatPlus/Docs/wiki/Debugging#on-the-fly-debug-output-for-individual-players) Use GitHub gists or http://pastebin.com/ or similar for huge logs, or attach a (zip-) file here. For image uploads, please prefer a site that allows viewing without JavaScript, you can upload file attachments for issues right here.

Debug log: https://pastebin.com/fD07rMqc
Contains debug log of me running over a platform with blocks above the ice, rubberbanded four times.

**Comment by asofold** [comment]
My ice-lane is just 23 blocks long :(. 

The ice stuff is a really old workaround  - i should model it more realistically. Then again it'd be better to also model other type of blocks (underneath) in a way, so we can estimate if acceleration+fricition is done right there.

**Comment by RoboMWM** [comment]
~~Duplicate of #350 (albeit this one follows a template with info)~~ Hmm, this one doesn't mention height though.

**Comment by asofold** [comment]
Ice remains a bigger topic, it does need better modeling. This probably should come alonngside with modeling friction and acceleration with touching the ground in general, i.e. also removing/replacing old bunny hop modeling.
